docs/USER-DOCUMENTATION.md

```diff
@@ -215,3 +217,4 @@ platforms.
 [etcher-cli]: https://github.com/resin-io/etcher/blob/master/docs/CLI.md
 [windows-usb-tool]: https://www.microsoft.com/en-us/download/windows-usb-dvd-download-tool
 [rufus]: https://rufus.akeo.ie
+[windows-iot-dashboard]: https://developer.microsoft.com/en-us/windows/iot/Docs/GetStarted/rpi3/sdcard/stable/GetStartedStep1.htm
```

Comment:
Looks like https://developer.microsoft.com/en-us/windows/iot/downloads might be a better "more generic" link?